website/docs/guides/duckdb-qs.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Quickstart for dbt Core using DuckDB
+id: duckdb
+description: "Learn to use dbt Core using DuckDB."
+hoverSnippet: "Learn to use dbt Core using DuckDB."
+platform: 'dbt-core'
+icon: 'duckdb-seeklogo'
+level: 'Beginner'
+hide_table_of_contents: true
+tags: ['dbt Core','Quickstart']
+---
+
+<div style={{maxWidth: '900px'}}>
+
+## Introduction
+
+In this quickstart guide, you'll learn how to use dbt Core with DuckDB, enabling you to get set up quickly and efficiently. [DuckDB](https://duckdb.org/) is an open-source database management system which is designed for analytical workloads. It is designed to provide fast and easy access to large datasets, making it well-suited for data analytics tasks. 
+
+
+This guide will demonstrate how to: 
+
+- [Create a virtual development environment](/docs/core/pip-install#using-virtual-environments) using a template provided by dbt Labs.
+   - This sets up a fully functional dbt environment with an operational and executable project. The codespace automatically connects to the DuckDB database and loads a year's worth of data from our fictional Jaffle Shop café, which sells food and beverages in several US cities.
+   - For additional information, refer to the [README](https://github.com/dbt-labs/jaffle_shop_duckdb/blob/duckdb/README.md) for the Jaffle Shop template. It includes instructions on how to do this, along with animated GIFs.
```

Comment:
Should these two be indented? I don't think they are referring to the virtual environment above it.

Reply:
Like setting up a venv does not give you access to duckdb automatically

Reply:
Also, I thought I was supposed to open the duckdb repo and read through, which wasn't necessary at this time. I think maybe rephrasing the last bullet as something like "we are going to run through the steps outlined in the jaffle_shop_duck_db repository, but if you want to dig into the underlying code further, you can find it there"